src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -1071,75 +1079,82 @@ SolidMechanicsLagrangianFEM::
 {
   GEOSX_MARK_FUNCTION;
 
-  MeshLevel const & mesh = domain.getMeshBody( 0 ).getMeshLevel( 0 );
-  NodeManager const & nodeManager = mesh.getNodeManager();
+  real64 totalResidualNorm = LvArray::NumericLimits< real64 >::max;
```

Comment:
??

Reply:
we need to do something with the residual when we have multiple bodies. I am taking the max across all mesh bodies.